docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.7'
+
+services:
+
+  database:
+    image: mongo:latest
+    restart: always
+    container_name: database
+    networks:
+      - local
+
+  node:
```

Comment:
`frontend` would be a more descriptive name.

Reply:
Its not the frontend container
Actually complete backend and frontend runs in same application.
We can make separate containers once backed apis and react frontend is ready

Reply:
It's also not the `node` container, that's just one of the technologies used in developing it. Perhaps `server` is a better word?

Since things are gonna change soon. I don't need this change, so turning it into an optional `nit`.

Reply:
We can rename the container as 'server' in subsequent builds to improve clarity and maintain consistency.